src/wp-includes/blocks.php

```diff
@@ -822,12 +828,18 @@ function serialize_block( $block ) {
  * parsed blocks.
  *
  * @since 5.3.1
+ * @since 6.4.0 The `$callback` parameter was added.
  *
- * @param array[] $blocks An array of representative arrays of parsed block objects. See serialize_block().
+ * @param array[] $blocks   An array of representative arrays of parsed block objects. See serialize_block().
+ * @param string  $callback Optional. Callback to run on the blocks before serialization. Default null.
```

Comment:
Nit, it cane be `string` or `null` now:
```suggestion
 * @param string|null  $callback Optional. Callback to run on the blocks before serialization. Default null.
```

Reply:
I just realized it can be more than that -- e.g. in the test it's an `array` 😬 (since that's how we specify a static class method).

Can I do `callable|null`? 🤔

Reply:
Yes, I think it's how it's done in other places. Good catch on top of my catch 😄

Reply:
d5781ad268